trpc/router/goal.ts

```diff
@@ -392,6 +463,15 @@ export const goal = router({
                     data: {
                         id, // this is hack to force updatedAt field
                         [kind]: { [connectionMap[String(direction)]]: connection },
+                        history: {
+                            create: {
+                                activityId: ctx.session.user.activityId,
+                                subject: 'dependencies',
+                                action: direction ? 'add' : 'remove',
+                                // FIXME: scoped id of deps goal
```

Comment:
It's not required. We are using shortId just for view and user navigation

Reply:
i'm forgot drop this comment :)